app/components/person/search-form.js

```diff
@@ -110,12 +126,6 @@ export default class SharedPersoonPersoonSearchFormComponent extends Component {
     } catch (e) {
       this.error = true;
     }
-    if (this.args.searchElected && this.args.bestuursperiode) {
-      return this.verkiezingService.getPeopleThatAreElected(
```

Comment:
This `getPeopleThatAreElected` is now only used with an array of just one person. So I would suggest renaming that function that something that makes more sense like `isElected` and simplify the logic a bit.

Reply:
makes sense!